client/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "quipucords-ui",
+  "version": "0.0.0",
```

Comment:
Could sync this with the quipucords version `0.0.1`, but that's no big deal.

Reply:
Sure can! Updating then I'll squash the commits down.